### DISCUSSION TITLE
[Enhancement]: keeping support for helm in non OCI repositories

### DISCUSSION THREAD

**Discussion by venkatamutyala** [strimzi/strimzi-kafka-operator#8913] [General]
### Related problem

Per the last release 0.36.0:

```The Helm Chart repository at https://strimzi.io/charts/ is now deprecated.
Please use the Helm Chart OCI artifacts from our [Helm Chart OCI repository instead](https://quay.io/organization/strimzi-helm).
```

This represents a huge issue for cluster admins/operators. Helm OCI support isn't fully embraced by all helm tools (ex. ArgoCD). It would be great if we could continue support for at least another 12 months or longer so that everyone can safely cut over to OCI. Until the main helm project deprecates the legacy registry I don't think this project should either.

Not sure what the timeline was for a full removal but just wanted to advocate for this earlier than later.

### Suggested solution

Keep publishing to both helm repositories until all major helm tools support OCI fully or until the helm project deprecates support for non-OCI itself

**Comment by scholzj** [comment]
As said in the 0.36 release notes. The old Helm chart repository will remain to exist for some time. But it is manually maintained whcih often leads to issues and errors and it is there fore not reliable. So you should migrate your tooling to the OCI Helm Charts or to use the Helm Chart binary directly from the GitHUb release page which are fully automated.

**Comment by rouke-broersma** [comment]
@venkatamutyala which tools do not support helm OCI? You mention ArgoCD, but ArgoCD supports helm OCI.

**Reply by venkatamutyala** [reply]
I tried converting some apps to OCI earlier in the year with ArgoCD. I ran into weird issues where the standard (non-OCI) helm repository was generally better supported. 

A few examples I was able to dig up from current/open GitHub issues:
https://github.com/argoproj/argo-cd/issues/13574
https://github.com/argoproj/argo-cd/issues/15344
https://github.com/yyzxw/argo-cd/commit/ae3caea2c232b3b2e26e57a658550e3a5349275c
https://github.com/argoproj/argo-cd/issues/12436
https://github.com/argoproj/argo-cd/issues/11709
https://github.com/argoproj/argo-cd/issues/10823
